### PR TITLE
<added CSS, and star ratings>

### PR DESCRIPTION
**Pull request by ValeriGuerrero**

### CODE REVIEW

File: src/AddTodoForm.jsx [hunk 1]
```diff
@@ -14,21 +16,25 @@ const AddTodoForm = ({ onAddTodo }) => {
...
             <>
                 <InputWithLabel
                     todoTitle={todoTitle}
+                    className='title'
                     handleTitleChange={handleTitleChange}
                 >
-                    Title
+                    Book Title
                 </InputWithLabel>
+                <RatingReview rating={rating} setRating={setRating} /> {/* Rating input */}
```
Comment (slamflipstrom) [inline src/AddTodoForm.jsx:34]: Nice experimenting here with other UI elements! Make sure you take some time to look at the react-rating-stars code to ensure that you understand what you're including in your codebase here. Do you think you could build your own rating component? I'm sure you could ;)

File: src/App.css [hunk 2]
```diff
@@ -1,16 +1,24 @@
+body{
+  background-color: cadetblue;
+  color: whitesmoke;
+  font-family: fantasy;
+  padding: 0;
+  margin: 0;
+}
+
+.booksLink{
+  position: relative;
```
Comment (slamflipstrom) [inline src/App.css:10]: this element should have a default position of `relative` so declaring this here in CSS is redundant. Try removing and see if that changes anything.

File: src/App.css [hunk 3]
```diff
@@ -40,3 +48,342 @@
...
+    background-color: rgb(221, 241, 221);
+    color: #000;
+    height: 25px;
+    font-weight: bold;
+    font-size: 15px;
+    align-items: baseline;
+    padding: 5px;
+    margin: 5px;
+  }
+}
```
Comment (slamflipstrom) [inline src/App.css:115]: This is a good start on media queries!

File: src/App.css [hunk 4]
```diff
@@ -40,3 +48,342 @@
...
+
+
+  /* Styles for tablets */
+@media (min-width: 481px) and (max-width: 768px){
```
Comment (slamflipstrom) [inline src/App.css:119]: I don't see styles changing at these viewport widths. Are you sure your media queries are working here?

File: src/App.css [hunk 5]
```diff
@@ -40,3 +48,342 @@
...
+  .addButton{
+    background-color: rgb(221, 241, 221);
+    color: #000;
+    height: 25px;
+    font-weight: bold;
+    font-size: 15px;
+    align-items: baseline;
+    padding: 5px;
+    margin: 5px;
+  }
```
Comment (slamflipstrom) [inline src/App.css:388]: You have a ton of styles in the stylesheet, some of which are redundant. I'd recommend considering a mobile-first styling approach here instead which would likely allow you to reduce much of this code. 

TLDR: build for the smallest screen size first and then scale media queries out in terms of width from there. You'll find this to be simpler and much less verbose.

https://www.freecodecamp.org/news/taking-the-right-approach-to-responsive-web-design/

**Review by slamflipstrom — APPROVED** [review]
I left some feedback for you to consider, but overall this code looks solid! Nice work, Valeri!
